cli/cmd/compliance_gcp.go

```diff
@@ -336,3 +379,37 @@ type cliComplianceIDAlias struct {
 	ID    string `json:"id"`
 	Alias string `json:"alias"`
 }
+
+type gcpAccount struct {
+	ProjectID      string
+	OrganizationID string
+}
+
+func extractGcpAccounts(response api.GcpIntegrationsResponse) []gcpAccount {
+	var gcpAccounts []gcpAccount
+
+	for _, gcp := range response.Data {
+		// if organization account, fetch the project ids
+		if gcp.Data.IDType == "ORGANIZATION" {
+			gcpAccounts = append(gcpAccounts, getGcpAccounts(gcp.Data.ID)...)
+		} else if !containsDuplicateProjectID(gcpAccounts, gcp.Data.ID) {
+			gcpIntegration := gcpAccount{OrganizationID: "n/a", ProjectID: gcp.Data.ID}
+			gcpAccounts = append(gcpAccounts, gcpIntegration)
+		}
+	}
+
+	sort.Slice(gcpAccounts, func(i, j int) bool {
+		return gcpAccounts[i].ProjectID < gcpAccounts[j].ProjectID
```

Comment:
I am not sure about this sorting mainly because, lets say there are two orgs, and each org has 50 projects, we would sort only by project and that will mix the grouping of the orgs. From a UX perspective, is that what we want¿?

In best case scenario, we should find a way to double sort, by org and then by projects.